client/dtmgrpc/dtmgimp/utils.go

```diff
@@ -77,7 +77,7 @@ func TransInfo2Ctx(ctx context.Context, gid, transType, branchID, op, dtm string
 
 // Map2Kvs map to metadata kv
 func Map2Kvs(m map[string]string) []string {
-	kvs := []string{}
+	kvs := make([]string, 0, len(m)<<1)
```

Comment:
len(m) << 1 is faster than len(m) * 2. But I still suggest to use  len(m) * 2 because it is more readable.

Reply:
done!